AdAway/src/main/res/values-de/strings.xml

```diff
@@ -68,99 +68,97 @@
+    <string name="apply_symlink_successful_title">Symlink-Erstellung erfolgreich</string>
+    <string name="apply_symlink_fail">Symlink konnte von Android nicht erstellt werden. Dies scheitert meist an einem \"Feature\" namens S-ON auf HTC-Handys! \n
+    \n
+  Eine Lösung besteht darin, das Telefon in den Wiederherstellungsmodus zu starten und den Symlink dort mit \'ln -s / data/ data/hosts/system/etc/hosts\' zu erstellen. \n
+    \n
+  Wenn dies nicht funktioniert, suchen Sie im Internet nach S-OFF und Ihrem Telefon.</string>
+    <string name="apply_symlink_fail_title">Symlink-Erstellung fehlgeschlagen</string>
+    <string name="apply_help">Bitte lesen Sie die Hilfe für weitere Informationen.</string>
+    <string name="apply_apn_proxy">Ein APN-Proxy ist auf Ihrem Android eingestellt! AdAway funktioniert nicht zuverlässig in mobilen Netzwerken wie 3G. Sie können diesen Proxy deaktivieren, indem Sie zum ausgewählten APN wechseln (unter Android 4: Drahtlos und Netzwerke -> Mehr ... -> Mobile Netzwerke -> Zugangspunktnamen) und den Wert im Proxy-Feld entfernen.</string>
     <string name="apply_apn_proxy_title">APN-Proxy eingestellt!</string>
-    <string name="no_connection">Die Internetverbindung funktioniert nicht.</string>
+    <string name="no_connection">Die Verbindung zum Internet funktioniert nicht.</string>
     <string name="no_connection_title">Keine Verbindung</string>
     <string name="download_dialog">Wird heruntergeladen …</string>
     <string name="apply_dialog">Wird angewendet …</string>
     <string name="apply_dialog_lists">Negativ- und Positivliste werden angewendet</string>
-    <string name="apply_dialog_hostnames">Hosts-Dateien werden verarbeitet und zusammengefügt</string>
-    <string name="apply_dialog_hosts">Hosts-Datei wird erstellt</string>
+    <string name="apply_dialog_hostnames">Analysiere und führe Hostdateien zusammen</string>
+    <string name="apply_dialog_hosts">Erstelle Host-Datei</string>
     <string name="apply_dialog_apply">Neue Hosts-Datei im System aktivieren</string>
-    <string name="apply_fail">Das Ändern der Hosts-Datei auf Ihrem System ist fehlgeschlagen!\n
-  Versuchen Sie bitte die Hosts-Datei, in den Einstellungen, auf /data/data/hosts zu ändern.</string>
+    <string name="apply_fail">Anwenden der Host-Datei auf Ihr System fehlgeschlagen! Versuchen Sie, die Zielhostdatei in den Einstellungen in /data/data/hosts zu ändern.</string>
     <string name="apply_fail_title">Anwendung fehlgeschlagen</string>
-    <string name="revert_successful">Die Änderungen wurden rückgängig gemacht.\n
-  Damit alle Änderungen wirksam werden, muss das System eventuell neu gestartet werden.\n
-  Möchten Sie neu starten?\n
-  (Um Datenverlust zu vermeiden, vergewissern Sie sich, dass keine Anwendung gerade die SD-Karte verwendet!)</string>
+    <string name="revert_successful">Sie haben die Standard-Hosts-Datei wiederhergestellt. Es wird empfohlen, Android Neu zu Starten, damit die Änderungen wirksam werden. \n
```

Comment:
Again that `neu starten`. I hope I am not wrong on this one, but it just seems wrong.